index.html

```diff
@@ -188,14 +188,14 @@
       </h3>
 
 
-      <p>Arabic script uses Arabic alphabet, diacritics, numbers, punctuations and symbols, and
-      control characters. Appendix <a href="#characters-tables"></a> lists these characters.</p>
+      <p>Arabic script uses Arabic alphabet, which consists mostly of letters and diacritics,
```

Comment:
I think better: The Arabic script uses the Arabic alphabet,

Reply:
Hmm, there are all sorts of problem here.

The start of the paragraph sounds  rather redundant, but i think it's actually incorrect. I don't think an alphabet includes digits and punctuation.  Furthermore, the 'Arabic alphabet' in my mind is different from the 'Persian alphabet', which is different from the Kashimiri or Uighur alphabets.  

Also, what does 'mostly' qualify?  Is is letters and diacritics only, or all the things in the list?  If the latter, what else does it consist of?

And then there's a mismatch between what this para says is in the table at the end and what's actually there, in that only two sets of digits are listed in the table.

Also, it's not just joining characters that are represented in the table.

How about simply:

Appendix ... lists characters used for the Arabic and Persian languages. Characters used for these languages include letters and diacritics, three sets of digits (depending on the region), punctuation (some common and some specific to the script), symbols, and Unicode formatting characters.

Reply:
Agreed. Updated the text based on your comment.